std/file.d

```diff
@@ -402,24 +402,24 @@ version (Posix) private void[] readImpl(scope const(char)[] name, scope const(FS
 
 
 version (Windows) private void[] readImpl(scope const(char)[] name, scope const(FSChar)* namez,
-                                          size_t upTo = size_t.max) @safe
+                                          size_t upTo = size_t.max) @system
```

Comment:
Since the posix version is @trusted, I would rather have this function be @trusted also.

Reply:
@RazvanN7, I changed it. Now it seems to pass the tests.